_rules/SC1-1-1-valid-duplicate-accessible-name-for-image.md

```diff
@@ -0,0 +1,78 @@
+---
+name: Valid use of duplicate accessible names for images
+description: |
+	This rule checks that there is a valid reason for having a set of images with identical accessible names.
+
+success_criterion:
+- 1.1.1 # Non-text Content
+
+test_aspects
+- DOM Tree
+- CSS Styling
+
+authors:
+- Bryn Anderson
+---
+
+### Applicability
+
+The rule applies to any set of HTML or SVG element with the semantic role of `img`, exposed to assistive technologies, that have an identical [accessible name]().
+
+### Expectation
+
+Each test target has an identical purpose.
+
+**Note**: Leading and trailing whitespace and difference in case sensitivity should be ignored.
+
+## Assumptions
+
+*There are currently no assumptions*
+
+## Accessibility support
+
+There is a known combination of a popular browser and assistive technology that does not by default support `title` as an accessible name.
+
+## Background
+
+## Test Cases
+
+### Pass example 1
+
+Image A and Image B are different files but have an identical purpose stated in the alt attribute and serve the same purpose linking to the same destination.
+
+```html
+<a href="https://twitter.com/brynanders"> <img src="/twitter_largeicon.png" alt="My twitter feed"> </a>
+<a href="https://twitter.com/brynanders"> <img src="/twitter_smallicon.png" alt="My twitter feed"> </a>
+```
+
+### Pass example 2
+
+Div A and B serve the same purpose but use the different image files to display a different version of the same informative image. Both files have an identical accessible name stated in the aria-label attribute.
+
+```html
+<div id="A" role="img" aria-label="blue sky" style="background-image:url('background1.jpg')"> </div>
+<div id="B" role="img" aria-label="blue sky" style="background-image:url('background2.jpg')"> </div>
+```
+
+### Failure example 1
+
+Image A and Image B are different files, have identical alt attributes but go to different destinations. 
+
+```html
+<a href="https://twitter.com/brynanders"> <img src="/twitter_largeicon.png" alt="Twitter"> </a>
+<a href="https://twitter.com"> <img src="/twitter_smallicon.png" alt="twitter"> </a>
+```
```

Comment:
I understand the example, but it mixes two elements : link and image.

If we want to keep that example, I think it would be necessary to add in the description, that the links both have the *same context*, so we can consider the two identical. 

In fact, in that example,  I think we don't evaluate the accessible name of the *image*, but of the *link*, and the accessible name of a link can be evaluate using the context (level AA [2.4.4 Link Purpose (In Context)](https://www.w3.org/TR/2008/REC-WCAG20-20081211/#navigation-mechanisms-refs) : see for example the [H78 technique (using enclosing paragraph)](https://www.w3.org/TR/WCAG-TECHS/H78.html), or [H79 (using table context)](https://www.w3.org/TR/WCAG-TECHS/H79.html) etc...  So, it's not because two links has the same accessible name (made of the alt attribute of the image in our eample) that they are the same, because of the context they may be part of.

Reply:
@audreymaniez interesting. Are you suggesting that [functional](https://www.w3.org/WAI/tutorials/images/functional/) images should be non-applicable because they would be covered by #220 ?

That would leave [informative](https://www.w3.org/WAI/tutorials/images/informative/), [complex](https://www.w3.org/WAI/tutorials/images/complex/) and groups of [images](https://www.w3.org/WAI/tutorials/images/groups/) applicable.

Reply:
> That would leave informative, complex and groups of images applicable.

Well, in fact, I think that any kind of images you list (informative, complex...) become innaplicable if the image is included in a link. At the moment, an image is included in a link (complex or not), its alt attribute is not used anymore to describe the image but to give the link a name. And even more : an image that can be evaluated as a decorative one, must have an alt text if its included in a link.

 So yes, I think that images, both within a link, with the same alt attribute and the same context, but that convey to different destination is a failed example for the test about identical links #220.